docs/api/io/v3/vm/folders.rst

```diff
@@ -0,0 +1 @@
+.. automodule:: tenable.io.v3.vm.folders
```

Comment:
I think this should be tenable.io.v3.vm.folders.api

Reply:
Updated